bazel/google_cloud_cpp_deps.bzl

```diff
@@ -50,12 +50,12 @@ def google_cloud_cpp_deps():
     if "com_github_grpc_grpc" not in native.existing_rules():
         native.http_archive(
             name = "com_github_grpc_grpc",
-            strip_prefix = "grpc-1.10.0",
+            strip_prefix = "grpc-1.14.0",
             urls = [
-                "https://mirror.bazel.build/github.com/grpc/grpc/archive/v1.10.0.tar.gz",
-                "https://github.com/grpc/grpc/archive/v1.10.0.tar.gz",
+                "https://github.com/grpc/grpc/archive/v1.14.0.tar.gz",
```

Comment:
You've changed the sort order here: `mirror.bazel.build` used to be first; does it matter? Is one faster than another?

Reply:
The mirror does not have v1.14.0.tar.gz (yet). I am talking to the folks that update that mirror, but it is a manual process apparently.

Reply:
Per offline conversation, the mirror now has it, so you can invert the order to make the diff smaller. Up to you.